spec/xtrabackuprb_spec.rb

```diff
@@ -0,0 +1,88 @@
+require 'spec_helper'
+
+describe 'osl-mysql::xtrabackuprb' do
+  ALLPLATFORMS.each do |pltfrm|
+    context "on #{pltfrm[:platform]} #{pltfrm[:version]}" do
+      cached(:chef_run) do
+        ChefSpec::SoloRunner.new(pltfrm).converge(described_recipe)
+      end
+      it do
```

Comment:
You need to add some stubs here so that rspec runs. This also needs to happen on the backup recipe and might make sense to have this as a shared_context so you don't duplicate code.